source/Service.js

```diff
@@ -421,6 +430,7 @@ class Service extends EventEmitter {
      * @memberof Service
      */
     async queryJobs(query = {}, { limit = Infinity, sort = "created", order = "desc" } = {}) {
+        query.archived = archived => archived === false || archived === undefined;
```

Comment:
Could we move this below the `!this._initialised` check? And perhaps it'd be good to add support *for* querying archived jobs. Perhaps adding a check to the `query` property like `query.archived = query.archived || ...`?

Reply:
True, I was thinking if we should be so strict on the query. I changed it, now I need to make a small change to the UI so it only queries for non-archived jobs.